highfidelity/hifi
# Use baked default skybox

Bake default skybox to save ~1s on startup time.

## Test Plan:
1) Launch interface
2) Make sure you see the default skybox
